File: source/code/nodejs/authentication/get_default_relay_state_with_extras.js
application.getSamlPolicy({ expand: 'serviceProvider' }, function (err, samlPolicy) {
  if (err) {
    return console.log(err);
  }

  var endpoint = samlPolicy.serviceProvider.href + '/defaultRelayStates';

  var options = {
    'organization': {
      'nameKey': 'my-org'
    }
  };

  client.createResource(endpoint, options, function(err, response){
    console.log(err, response.defaultRelayState);
  });
});

File: source/code/ruby/authentication/get_default_relay_state_with_extras.rb
drs = application.saml_policy.service_provider.default_relay_states.create(
  callback_uri: 'https://org1.myapp.com',
  organization: {
    name_key: 'org1'
  },
  state: 'iamastate'
)

# This is also possible:
drs = application.saml_policy.service_provider.default_relay_states.create(
    callback_uri: 'https://org1.myapp.com',
    organization: organization,
    state: 'iamastate'
)
